PresentScreenings/Utilities/ScreeningsPlanner.cs

```diff
@@ -40,7 +40,9 @@ public void MakeScreeningsPlan(string filmFan)
             while (rating.IsGreaterOrEqual(FilmRating.LowestSuperRating))
             {
                 // Select films with this rating.
-                var films = ScreeningsPlan.Films.Where(f => ViewController.GetMaxRating(f).Equals(rating)).ToList();
+                var films = ScreeningsPlan.Films
+                    .Where(f => ViewController.GetMaxRating(f).Equals(rating) && ViewController.FilmScreenings(f.FilmId).Count > 0)
```

Comment:
Please consider to create a property of class `Film` to replace `ViewController.FilmScreenings(f.FilmId).Count > 0`.

Reply:
Done.